test/modules/EdgeHubRestartTester/docker/linux/arm64v8/Dockerfile

```diff
@@ -1,5 +1,9 @@
-ARG base_tag=1.0.6.15-linux-arm64v8
-FROM azureiotedge/azureiotedge-module-base:${base_tag}
+ARG base_tag=3.1.19-bionic-arm64v8
```

Comment:
Eventually, we will need to update these tags. 

We really should move to env_file: https://docs.docker.com/compose/compose-file/compose-file-v3/#env_file

$ docker run --env-file=env_file_name ... .... ?

Should work..?

Reply:
Really, this is the kind of thing that could be automated -- most of these Dockerfiles are very similar, with like 2 lines that are different (module/DLL name). 

If nothing else, we could have per-module templates? Cost-Benefit conversation will need to be had, of course.